tests/Jira/ApiTest.php

```diff
@@ -14,116 +15,164 @@
 class ApiTest extends \PHPUnit_Framework_TestCase
 {
 
+	const ENDPOINT = 'http://jira.company.com';
+
+	/**
+	 * Api.
+	 *
+	 * @var Api
+	 */
+	protected $api;
+
 	/**
-	 * Tests that any trailing slash in the endpoint url is removed before being stored in the object state
+	 * Credential.
+	 *
+	 * @var AuthenticationInterface
 	 */
-	public function testSetEndpointTrailingSlash()
```

Comment:
Then that's why I've got error from GitHub, when tried to reply to that
comment on website. :)

Decided to reply over email.